ci-operator/config/oVirt/csi-driver-operator/OWNERS

```diff
@@ -0,0 +1,10 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- bennyz
+- rgolangh
```

Comment:
this is CI, I would also add Gal Zaidman to reviewers / approvers

Reply:
done